package.json

```diff
@@ -16,7 +16,10 @@
   "homepage": "https://github.com/commercetools/eslint-config",
   "peerDependencies": {
     "eslint": ">=2.0",
-    "eslint-plugin-react": ">=4.0"
+    "eslint-config-airbnb": ">=7.0",
+    "eslint-plugin-jsx-a11y": ">=1.0",
```

Comment:
You're right, we don't actually need it. It's a peer dependency of `eslint-config-airbnb`.